rules/ruleset.go

```diff
@@ -441,12 +445,43 @@ type RuleSet interface {
 	CutoverNanos() int64
 
 	// TombStoned returns whether the ruleset is tombstoned.
-	TombStoned() bool
+	Tombstoned() bool
 
 	// ActiveSet returns the active ruleset at a given time.
 	ActiveSet(timeNanos int64) Matcher
 
+	// Schema returns the schema.Ruleset representation of this ruleset.
 	Schema() (*schema.RuleSet, error)
+
+	// Tombstone tombstones this ruleset and all of its rules.
+	Tombstone(time.Duration) error
+
+	// AppendMappingRule creates a new mapping rule and adds it to this ruleset.
+	AddMappingRule(string, map[string]string, []policy.Policy, time.Duration) error
```

Comment:
Will add names to the params in the interface for now, am going to introduce those structs in the next PR.